variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "prefix" {
+  description = "The prefix used for all resources in this environment"
+  default     = "ani-dev"
+}
+
+variable "location" {
+  description = "The Azure location where all resources in this deployment should be created"
+  default     = "uksouth"
+}
+
+variable "github_oauth_client_id" {
+  description = "Github oauth client id"
+}
+
+variable "github_oauth_client_secret" {
+  description = "Github oauth client secret"
+}
+
+variable "secret_key" {
+  description = "Flask app secret key"
+  default     = "super_secret_key"
```

Comment:
Here we're generally not being too cautious with this value, but worth noting that we should really be treating it as a secret!